product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx

```diff
@@ -104,14 +112,14 @@ __OUTPUT__
 ...
 ```
 
-This table has a "reportsto" field - that means this is a heirarchical reporting structure, with some employees reporting to
+This table has a "reportsto" field - that means this is a hierarchical reporting structure, with some employees reporting to
```

Comment:
It's not immediately clear to me what "reportsto" means. Maybe:

> This table has a "reportsto" field  which defines a hierarchical reporting structure . . .

That defines the field along the way.